### DISCUSSION TITLE
How can we copy data from formula column to a non-formula column?

### DISCUSSION THREAD

**Discussion by hexxamillion** [nocodb/nocodb#9729] [General]
One feature that I dislike about NocoDB is that formula fields do not create actual fields in the database. How can I copy data from a formula field to a different field so that I can perform db queries against it? Basically copy a column to another. I really just need to do this one time for the existing data. I have triggers setup in the db for new inserts and updates.

**Comment by dstala** [comment]
@hexxamillion Type conversion for formula field is not supported currently. This request is in our roadmap though, you can help prioritise it by upvoting here

https://github.com/nocodb/nocodb/issues/7639
